src/app/workbasket/workbasket.component.ts

```diff
@@ -94,6 +98,14 @@ export class WorkbasketComponent implements OnInit {
         });
 
     this.scrollToTop();
+
+    this.router.navigate(['/list/case'], {
```

Comment:
Do we have a test case asserting this action?

Reply:
Added now